src/js/components/products/listItem.tsx

```diff
@@ -32,6 +33,14 @@ const ProductItem = ({ item }: { item: Product }) => {
         >
           <div className="slds-text-title">
             {t('Version {{version}}', { version: label })}
+            {item.tags
+              ? item.tags.map((tag) => (
```

Comment:
I think the rendering of tags should also depend on the value of `window.GLOBALS.SITE.show_product_tags`